test/bootstrap_radio_button_test.rb

```diff
@@ -1,4 +1,6 @@
 require 'test_helper'
+require 'nokogiri'
+require 'equivalent-xml'
```

Comment:
The only reason those `require` statements are there is that's the file I arbitrarily chose to initially do the work. What I checked in yesterday I hadn't even reviewed myself. I didn't realize how quickly you were going to take the commit. :-) I agree the `require`s should go in `test_helper.rb`.